dotnet/test/common/Environment/EnvironmentManager.cs

```diff
@@ -231,9 +236,15 @@ public void CloseCurrentDriver()
         {
             if (driver != null)
             {
-                driver.Quit();
+                try
```

Comment:
We are ignoring potential issues in `Quit()` method here.

Reply:
Yes, this needed to catch/ignore as well. Thanks.